source/scss/_library/_objects.buttons.scss

```diff
@@ -8,8 +8,7 @@ input[type="submit"] {
   display: inline-flex;
```

Comment:
no `appearance: none;`?

Reply:
Hmm, that's usually in there. I'll add it!